Home Screen / Character Hover

Konsttakis opened the pull request:
Changed character hover so when one is hovered over the others don't change in size.

chavi362 reviewed (commented):
First of all thank you very much!
I didn't understand what you have change I still see that all the characters change the size.

chavi362 reviewed (changes requested):
@Konsttakis I see that the characters still change size. Maybe you didn't upload the right code?
Can you please check this?
Thank you so much!
Best regards,
Chavi

Konsttakis commented:
> First of all thank you very much! I didn't understand what you have change I still see that all the characters change the size.

Thank you for reviewing my changes and sorry for the mishap. I think now the changes are implented and take effect.

chavi362 reviewed (changes requested):
The all characters still change size when hovering on one

Konsttakis commented:
> The all characters still change size when hovering on one

When you hover over the first one for example the others don't change in size and neither does the background. Before, when you hovered over the first one for example the others also changed and moved so it seemed cluttered
